### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by ayumitk**
#98

### CODE REVIEW

File: .github/workflows/broken-links-action.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
         with:
-          args: --verbose --exclude-path ./node_modules/ "./**/*.js" "./**/*.ts" "./**/*.html" "./**/*.scss" "./**/*.json" "./**/*.md" "./**/*.vue"
+          args: --verbose --exclude-path ./node_modules/ "./**/*.js" "./**/*.ts" "./**/*.html" "./**/*.scss" "./**/*.json" "./**/*.md" "./**/*.vue" "./nuxt.config.ts"
```
Comment (ayumitk) [inline .github/workflows/broken-links-action.yml:22]: @niklabh I want to ignore `nuxt.config.ts` file. Is this correct?

Comment (ayumitk) [reply .github/workflows/broken-links-action.yml:22]: or should we check just inside `.output/public/` folder?

Comment (niklabh) [reply .github/workflows/broken-links-action.yml:22]: yes we can just check in .output/public/ folder

Comment (ayumitk) [reply .github/workflows/broken-links-action.yml:22]: @niklabh Could you set this up only in .output/public/ folder?

**Pull request closed without merging**